docs/source/usage/wm-widgets.rst

```diff
@@ -50,24 +50,27 @@ Add the following to :file:`~/.config/qtile/config.py`:
        ),
    ]
 
-.. _bar-usage:
+.. _lemonbar-usage:
 
-bar-aint-recursive
-==================
+lemonbar (formerly bar-aint-recursive)
+======================================
 
-To run the bar simply pipe the output of the binding script into ``bar`` and 
-specify appropriate options, for example like this::
+To run the bar simply start the binding script:
 
-    python /path/to/powerline/bindings/bar/powerline-bar.py | bar
+    python /path/to/powerline/bindings/lemonbar/powerline-lemonbar.py
 
-to run with i3, simply ``exec`` this in i3 config file::
+You can specify options to be passed to ``lemonbar`` after ``--``, like so:
```

Comment:
(BTW, since there are such problems with naming maybe there should be additional argument like `--command=lemonbar`? I doubt that all systems which have lemonbar have both `lemonbar` and `bar-aint-recursive` programs (symlinks?).)